src/Compilers/Lua/Test/Portable/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -1223,6 +1224,17 @@ public void SyntaxNormalizer_CorrectlyRewritesTypes(string input, string expecte
             AssertNormalizeCore(type, expected);
         }
 
+        [Fact]
+        [WorkItem(108, "https://github.com/LorettaDevs/Loretta/issues/108")]
+        public void SyntaxNormalizer_CorrectlyInsertsExpressionSpaces()
+        {
+            var tree = ParseAndValidate("print(1, 2)", s_luaParseOptions);
```

Comment:
It'd be better to add an input with messed up indentation here to ensure we're actually formatting the code and not just spitting out the code that gets input